protocols/manage/count.go

```diff
@@ -106,9 +106,11 @@ func (p *ProtocolCount) FuncPC() {
 		p.SendTo(p.Parent(), &NodeIsUp{})
 	}
 	if !p.IsLeaf() {
-		for _, c := range p.Children() {
-			dbg.Lvl3(p.Myself(), "sending to", c.Entity.Addresses, c.Id, p.timeout)
-			p.SendTo(c, &PrepareCount{Timeout: p.timeout})
+		for _, child := range p.Children() {
+			go func(c *sda.TreeNode) {
```

Comment:
Out of curiosity: why using go routines ? I don't mind except I prefer *not* using go routines where this is *not* necessary.